### DISCUSSION TITLE
Proxmox Backup Server - unprivileged but sees root disks... sort of

### DISCUSSION THREAD

**Discussion by SoTHISIsFitness** [tteck/Proxmox#3164] [Help Needed]
So I just spun up a PBS LXC using the script (thanks tteck!) and while poking around, noticed that PBS... within an unprivileged LXC... sees not only its own virtual hard drive... but also... ALL of the PVE host's actual physical disks! It sees them all as 'unused', and apparrently this odd situation is because it can see the /sys folder but it doesn't have the block devices from the /dev folder (saw that in [this thread in proxmox forums](https://forum.proxmox.com/threads/lxc-container-can-access-all-host-disks-but-shouldnt.129679/#post-568421))

So, question is: 
a.) is this going to cause problems or potential "oops you borked your server!" problems?
and
b.) either way, is there a way to modify the helper script for the PBS LXC to... idk... prevent it from seeing the PVE host's actual physical disks?

idk, it just feels like it shouldn't be that way and is begging for trouble, but that could just be me not knowing any better 🤷‍♂️

**Comment by Mellowlynx** [comment]
Your questions are pretty match answered on the forum you linked.
But yes, it might brick your server if you don't know what you are doing, but this is true for every OS/machine.

Also, did you see the staff's reaction?
 `PBS is not really intended to be run in an (unprivileged) container.`
 And have you thought about where/how you would get the backups if your node with the PBS LXC on it crashes, goes down, or whatever?

I have a separate machine for just PBS, so after reinstalling Proxmox and running the installation aftercare, I can add the PBS back and start restoring in minutes.
For testing, it's fine and a nice to have, but **please** test your backup solutions, before you really need it, to check if it works in a production setting.
